coopr-docs/docs/source/guide/superadmin/chef-solo-automator-plugin.rst

```diff
@@ -198,22 +211,77 @@ definition with the following parameters:
 
 Then simply add your service to a cluster template.
 
-
 Helper Cookbooks
 ================
 
-Coopr ships with several helper cookbooks.
+Coopr ships with several helper cookbooks. These are provided as convenience and are completely optional. This section provides a brief overview of each.
 
 
 **coopr_base**
 --------------
-This is a convenience cookbook which is intended to provide base functionality for all hosts provisioned by coopr.  It currently 
-does the following:
+This is a convenience cookbook which is intended to provide base functionality for all hosts provisioned by Coopr. The default ``base`` service definition runs
+the ``coopr_base::default`` recipe, which may include additional helper cookbooks. It currently does the following:
 
 	* run ``apt-get update`` (on Ubuntu hosts)
-	* include ``coopr_hosts::default`` (discussed below)
-	* include ``coopr_firewall::default`` (discussed below)
+        * optionally configure the Yum ``epel`` repository (on Rhel hosts)
+	* optionally include ``coopr_dns::default`` (discussed below)
+	* optionally include ``coopr_firewall::default`` (discussed below)
+	* optionally include ``coopr_hosts::default`` (discussed below)
+	* optionally include ``coopr_packages::default`` (discussed below)
 	* include ``ulimit::default`` to enable user-defined ulimits
+        * optionally runs the ``users`` and ``sudo`` cookbooks to add any users if a ``users`` data_bag resource is present
+
+To disable the configuring of the Yum ``epel`` repository on Rhel hosts, set the ``node['base']['use_epel']`` attribute to ``false``
+
+To disable the inclusion of the remaining helper cookbooks, set the following attributes. Though note that these cookbooks are designed to not take action unless
+specific attributes are added to your Coopr templates::
+
+    node['base']['no_dns'] = 'true'
+    node['base']['no_firewall'] = 'true'
+    node['base']['no_hosts'] = 'true'
+    node['base']['no_packages'] = 'true'
+
+**User Management**
```

Comment:
it's part of the coopr_base cookbook, so i think it should be a sub-section.  If there is a better format to use, let me know